dockertest/output/texttable.py

```diff
@@ -135,6 +135,7 @@ class TextTable(MutableSet, Sequence):
     _rows = None
 
     def __init__(self, table, columnranges=None, header=None, tabledata=None):
+        # pylint: disable=W0231
```

Comment:
W0231:137,4: TextTable.__init__: __init__ method from base class 'Sequence' is not called
    W0231:137,4: TextTable.__init__: __init__ method from base class 'MutableSet' is not called